astropy/io/fits/tests/test_fitscheck.py

```diff
@@ -19,11 +20,16 @@ def test_help(self):
         assert e.value.code == 0
 
     def test_version(self, capsys):
-        with pytest.raises(SystemExit) as e:
-            fitscheck.main(["--version"])
-            out = capsys.readouterr()[0]
-            assert out == f"fitscheck {version}"
-        assert e.value.code == 0
+        script = "fitscheck"
+        p = subprocess.Popen(
```

Comment:
What's the advantage of calling `subprocess.Popen()` instead of `subprocess.run()`?

Reply:
There is none, I will switch it to run.